docs/docs/known-issues.md

```diff
@@ -1,18 +1,34 @@
 
 
 ---
-* Currently, BigDL uses synchronous mini-batch SGD in model training. The mini-batch size is expected to be a multiple of **total cores** used in the job.
+* Currently, BigDL uses synchronous mini-batch SGD in model training. The mini-batch size is
+expected to be a multiple of **total cores** used in the job.
 
-* You may observe very poor performance when running BigDL for Spark 2.0 with Java 7; it is highly recommended to use Java 8 when building and running BigDL for Spark 2.0.
+* You may observe very poor performance when running BigDL for Spark 2.0 with Java 7; it is highly
+recommended to use Java 8 when building and running BigDL for Spark 2.0.
 
-* On Spark 2.0, please use default Java serializer instead of Kryo because of [Kryo Issue 341](https://github.com/EsotericSoftware/kryo/issues/341). The issue has been fixed in Kryo 4.0. However, Spark 2.0 uses Kryo 3.0.3. Spark 1.5 and 1.6 do not have this problem.
+* On Spark 2.0, please use default Java serializer instead of Kryo because of
+[Kryo Issue 341](https://github.com/EsotericSoftware/kryo/issues/341). The issue has been fixed in
+Kryo 4.0. However, Spark 2.0 uses Kryo 3.0.3. Spark 1.5 and 1.6 do not have this problem.
```

Comment:
Do you know if Spark 2.1 or 2.2 still have this problem?

Reply:
just check the master code. it still uses kryo 3.0.3